compiler/rustc_mir/src/interpret/eval_context.rs

```diff
@@ -64,7 +59,6 @@ impl SpanGuard {
         // This executes the destructor on the previous instance of `SpanGuard`, ensuring that
         // we never enter or exit more spans than vice versa. Unless you `mem::leak`, then we
         // can't protect the tracing stack, but that'll just lead to weird logging, no actual
-        // problems.
```

Comment:
This looks wrong, now the sentence has no end any more?

Reply:
Fixed!